lib/rubocop/cop/rspec/leaked_const.rb

```diff
@@ -0,0 +1,61 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that constants are not created in tests without cleanup.
+      #
+      # Prefer stub_const and anonymous classes.
+      #
+      # @example
+      #   # bad
+      #   describe MyClass do
+      #     class Foo
+      #       BAR = 1
+      #     end
+      #   end
+      #
+      #   # good
+      #   describe MyClass do
+      #     before do
+      #       stub_const('Foo::BAR', 1)
+      #     end
+      #   end
+      class LeakedConst < Cop
+        MSG = 'Opening a class to define methods can pollute your tests. '\
```

Comment:
We might be able to improve this message and explain why they pollute the test. Additionally, it might be nice to mention using `Class.new` since I (personally) think that's simpler than / preferable to using `stub_const`. I'm not sure how to put it succinctly yet though (also, i'm not saying this message is bad--it's not, just thinking we may be able to make it better).